src/utils/validators.ts

```diff
@@ -0,0 +1,20 @@
+export const isRequired = (inputName: string, value: string) => {
+    if (!value || value.length === 0) {
+        return `${inputName} is required.`
+    }
+    return true
+}
+
+export const isKebabCase = (inputName: string, value: string) => {
+    if (!/^[\w-.]+$/.test(value)) {
+        return `${inputName} can only contain letters, dashes, dots and underscores.`
```

Comment:
just a naming nitpick, this seems to be validating more than just kebab case, would something like "isValidKey" be more accurate?

Reply:
agreed. Will make the change